src/translations/en.json

```diff
@@ -1808,7 +1808,7 @@
         },
         "integrations": {
           "caption": "Integrations",
-          "description": "Manage integrations",
+          "description": "Manage integrations like Hue, Sonos, Z-Wave and Zigbee",
```

Comment:
Why these examples ? I personnaly use none of these.
With noob eyes that would mean these are configured and i don't know why ?
I would not name any intergration here.
"Add, remove and edit integrations" => Manage integrations...

Reply:
I am sympathetic to the argument that it's a little confusing to list integrations that you may or may not use.

I don't think "Manage integrations" or "Add, remove, and edit integrations" is helpful.

Any other ideas for trying to explain what an integration *is*, I'm all ears.

Reply:
I borrowed the string from the [Figma design mock for future config changes](https://www.figma.com/file/4LZqQZRynDZGTgA2rN35Te/Home-Assistant-config-mockup?node-id=0%3A1), which should address your concern here. It is now "Manage devices & services"